CloudFormationLab.yml

```diff
@@ -34,9 +33,8 @@
         Type: 'AWS::EC2::Subnet'
         Properties:
           VpcId: !Ref PubPrivateVPC
-          AvailabilityZone: us-east-1a
           CidrBlock: 172.31.3.0/24
-          MapPublicIpOnLaunch: false
+          MapPublicIpOnLaunch: false #makes it a private subnet
```

Comment:
Does not make it private subnet, just does not give public  on launch by default. You can still get public ip if you want

Reply:
So... need to find way to make public IP unavailable?

Reply:
no, false is correct here but comment is wrong. It does not make subnet private. Associating this subnet with routing table without internate gateway is what makes it private subnet. This simply says do not assign public facing IP